### PR TITLE
Create form to edit user and create profile

### LINKED ISSUE #38: Add Profile edit page
Users can sign up for accounts now but they need to be able to edit their details

### PR DESCRIPTION
**Pull request by rifferreinert**
closes #38 

This PR creates a page which allows new users to edit their information and create a profile

### CODE REVIEW

File: buddy_mentorship/settings/local.py [hunk 1]
```diff
@@ -128,7 +128,7 @@
 AUTH_USER_MODEL = "users.User"
 
 # Redirect to home URL after login (Default redirects to /accounts/profile/)
-LOGIN_REDIRECT_URL = "/"
+LOGIN_REDIRECT_URL = "/profile/"
```
Comment (ecxia) [inline buddy_mentorship/settings/local.py:131]: Not sure how I feel about this behavior - for example, think of case where user clicks "Requests" and gets sent to login. Shouldn't we try to send them back to requests? Not a blocking issue.

File: buddy_mentorship/templates/buddy_mentorship/edit_profile.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% extends 'buddy_mentorship/base.html' %}
+
+{% block title %}
+	Homepage
```
Comment (ecxia) [inline buddy_mentorship/templates/buddy_mentorship/edit_profile.html:4]: Update title

File: buddy_mentorship/views.py [hunk 3]
```diff
@@ -4,9 +4,12 @@
 from django.shortcuts import render, redirect
 from django.views.generic import ListView
 from django.contrib.auth.mixins import LoginRequiredMixin
+from django.views.generic.edit import FormView
+from django.urls import reverse
```
Comment (ecxia) [inline buddy_mentorship/views.py:8]: reverse doesn't get used; clean up import

File: buddy_mentorship/settings/local.py (hunk 1 above)
Comment (rifferreinert) [reply buddy_mentorship/settings/local.py:131]: That will still work, this is only when they hit login

**Pull request merged**